Source/Core/DolphinQt2/Settings/AudioPane.cpp

```diff
@@ -0,0 +1,227 @@
+  m_stretching_buffer_label = new QLabel(tr("?? ms"));
+  stretching_box->setLayout(stretching_layout);
+
+  m_stretching_buffer_slider->setMinimum(80);
+  m_stretching_buffer_slider->setMaximum(300);
+  m_stretching_buffer_slider->setTickInterval(5);
+
+  stretching_layout->addWidget(m_stretching_enable, 1, 1, 1, 3);
+  stretching_layout->addWidget(new QLabel(tr("Buffer Size")), 2, 1);
+  stretching_layout->addWidget(m_stretching_buffer_slider, 2, 2);
+  stretching_layout->addWidget(m_stretching_buffer_label, 2, 3);
+
+  m_main_layout = new QGridLayout;
+
+  m_main_layout->addWidget(dsp_box, 1, 1, 1, 3);
+  m_main_layout->addWidget(volume_box, 1, 4);
+  m_main_layout->addWidget(backend_box, 2, 1, 1, 4);
+  m_main_layout->addWidget(stretching_box, 3, 1, 1, 4);
+
+  setLayout(m_main_layout);
+}
+
+void AudioPane::ConnectWidgets()
+{
+  connect(m_backend_combo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
+          [this](int) { SaveSettings(); });
+  connect(m_volume_slider, &QSlider::valueChanged, [this](int) { SaveSettings(); });
+  connect(m_latency_spin, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged),
+          [this](int) { SaveSettings(); });
+  connect(m_stretching_buffer_slider, &QSlider::valueChanged, [this](int) { SaveSettings(); });
+  connect(m_dolby_pro_logic, &QCheckBox::toggled, this, &AudioPane::SaveSettings);
+  connect(m_stretching_enable, &QCheckBox::toggled, this, &AudioPane::SaveSettings);
+  connect(m_dsp_hle, &QRadioButton::toggled, [this](bool) { SaveSettings(); });
+  connect(m_dsp_lle, &QRadioButton::toggled, [this](bool) { SaveSettings(); });
+  connect(m_dsp_interpreter, &QRadioButton::toggled, [this](bool) { SaveSettings(); });
+}
+
+void AudioPane::LoadSettings()
+{
+  auto& settings = Settings::Instance();
+
+  // DSP
+  if (settings.GetDSPHLEEnabled())
+  {
+    m_dsp_hle->setChecked(true);
+  }
+  else
+  {
+    m_dsp_lle->setChecked(settings.GetDSPJITMode());
+    m_dsp_interpreter->setChecked(!settings.GetDSPJITMode());
+  }
+
+  // Backend
+  const auto current = settings.GetCurrentAudioBackend().toStdString();
+  for (const auto& backend : AudioCommon::GetSoundBackends())
+  {
+    m_backend_combo->addItem(tr(backend.c_str()));
+    if (backend == current)
+      m_backend_combo->setCurrentIndex(m_backend_combo->count() - 1);
+  }
+
+  OnBackendChanged();
+
+  // Volume
+  m_volume_slider->setValue(settings.GetVolume());
+  m_volume_label->setText(tr("%1 %").arg(settings.GetVolume()));
+
+  // DPL2
+  m_dolby_pro_logic->setChecked(settings.GetDPLEnabled());
+
+  // Latency
+  m_latency_spin->setValue(settings.GetLatency());
+
+  // Stretch
+  m_stretching_enable->setChecked(settings.GetAudioStretchEnabled());
+  m_stretching_buffer_slider->setValue(settings.GetAudioStretch());
+  m_stretching_buffer_slider->setEnabled(m_stretching_enable->isChecked());
+  m_stretching_buffer_label->setText(tr("%1 ms").arg(settings.GetAudioStretch()));
```

Comment:
That's not how it was done previously, previously not meaning Wx but instead all the other Qt settings for the most part. I can change this, but I think it should be changed across the board then, don't you think?